docs/out-of-tree-platforms.md

```diff
@@ -3,15 +3,19 @@ id: out-of-tree-platforms
 title: Out-of-Tree Platforms
 ---
 
-React Native is not only for Android and iOS - there are community-supported projects that bring it to other platforms, such as:
+React Native is not only for Android and iOS devices - there are partners-supported and community-supported projects that bring it to other platforms, such as:
 
-- [React Native Windows](https://github.com/Microsoft/react-native-windows) - React Native support for targeting Microsoft's Universal Windows Platform (UWP).
+**From Partners**
+
+- [React Native Windows](https://github.com/microsoft/react-native-windows) - React Native library for targeting Microsoft's Universal Windows Platform (UWP).
```

Comment:
Maybe we could simplify the description for each of these projects to remove unnecessary variations (e.g. what is a fork, what is a library).

> React Native for Microsoft's Universal Windows Platform (UWP).